pkgdb2client/cli.py

```diff
@@ -985,8 +985,6 @@ def main():
     pkgdbclient.insecure = arg.insecure
 
     if arg.bzurl != pkgdb2client.BZ_URL:
-        if not arg.bzurl.endswith('xmlrpc.cgi'):
-            arg.bzurl = '%s/xmlrpc.cgi' % arg.bzurl
```

Comment:
Yes, but if you don't specify it in the url, the class initialization in python-bugzilla already fix it.

import bugzilla
BZ=bugzilla.Bugzilla(url='https://bugzilla.mozilla.org')
BZ.url
Out[11]: 'https://bugzilla.mozilla.org/xmlrpc.cgi'

That been said, if BZ=bugzilla.Bugzilla(url='https://bugzilla.mozilla.org**/**') is used (with the ending **/**), it breaks, so best to keep it .

You can discard theses PR, thx